src/asciidoc/docs/about/contribute.adoc

```diff
@@ -0,0 +1,176 @@
+= Contributing to the Koji developer site
+:page-slug: contribute-koji-developers
+:page-banner: {imagesDir}/example.png
+
+The Koji for Developers documentation site is built and maintained by developers just like you.
+You too can help by contributing to the site *structure*, writing new *articles*, or *suggesting edits* to existing documentation.
+
+Anyone can open an issue about documentation, or contribute a change with a pull request to the `madewithkoji/docs` https://github.com/madewithkoji/docs[GitHub repository].
+
+link to code of conduct?
+
+include::../_includes/tip-discord-server.adoc[tags=*]
+
+== Editing the documentation
+
+The documentation is written in https://asciidoc.org/[Asciidoc] which is a lightweight markup language for simplifying the creation of documentation.
+All the AsciiDoc files that map to this site can be found in the repository under `src/asciidoc/`.
+
+Here's a quick example:
+
+[source, asciidoc]
+----
+= Koji for Developers
+Create your own templates for https://withkoji.com[Koji].
+
+
+== Use any framework
+
+* React
+* Vanilla JS
+
+[source,javascript]
+console.log('Hello World');
+----
+
+
+To see more and get a better understanding of the syntax visit the https://powerman.name/doc/asciidoc[AsciiDoc cheatsheet] and the https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/[AsciiDoc Syntax Quick Reference].
+
+We also have are a few extensions on AsciiDoc syntax added in to make the developer site ever better.
+
+=== Page Attributes
+Page attributes are used to add extra data related to a document.
+Add these to the top of your document, just under the document title.
+
+Page slug::
+	This corresponds to the page url and is used for linking between different pages. +
+	`:page-slug: example-page`
+Page description::
+	This is a short description of the content of the page and is used when sharing the link on social media and chat platforms. +
+	`:page-description: This page is an example`
+Page banner::
+	This references an image which is used as the thumbnail when sharing the link on social media and chat platforms. +
+	`:page-banner: \{imagesDir\}/example-page.png`
+
+NOTE: `\{imagesDir\}` refers to the image directory, which can be found in the repository at `static/images`
+
+TIP: If you have block images on your page you don't need to set the page banner, the first image will be used instead
```

Comment:
```suggestion
TIP: If the page contains block images, you don't need to set the page banner attribute. The first block image on the page is used, instead.
```
Also, wondering if we should combine this tip into the note above (maybe as two bullets in the same note). Just to avoid having back-to-back admonitions.